src/core_functions/scalar/date/date_part.cpp

```diff
@@ -393,8 +393,13 @@ struct DatePart {
 		static void Inverse(DataChunk &input, ExpressionState &state, Vector &result) {
 			D_ASSERT(input.ColumnCount() == 1);
 
-			UnaryExecutor::Execute<int64_t, timestamp_t>(input.data[0], result, input.size(),
-			                                             [&](int64_t input) { return Timestamp::FromEpochMs(input); });
+			UnaryExecutor::Execute<int64_t, timestamp_t>(input.data[0], result, input.size(), [&](int64_t input) {
+				auto ts = timestamp_t(input);
```

Comment:
`select epoch_ms(9223372036854775807);` now produces `infinity`, not sure if this is expected behavior?

Reply:
It now throws an error instead